okhttp-hpacktests/pom.xml

```diff
@@ -8,7 +8,7 @@
   <parent>
     <groupId>com.squareup.okhttp3</groupId>
     <artifactId>parent</artifactId>
-    <version>3.3.0-SNAPSHOT</version>
```

Comment:
That was unfortunate. Are there any direct (or indirect) impediments to running these with the normal build? I don't see it in the parent pom.xml.